kubernetes-elasticsearch-cluster/deploy-es.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+if (( $# != 1 ))
+then
+  echo "Usage: kubernetes-elasticsearch-cluster/deploy-es.sh <dataset>"
+  echo "  where <dataset> is the name of a directory in dataset_config/"
+  echo "Run this script from project root"
+  exit 1
+fi
+
+dataset=$1
```

Comment:
Similar to other scripts -- eg [deploy-indexer.sh](https://github.com/DataBiosphere/data-explorer-indexers/blob/master/bigquery/deploy/deploy-indexer.sh#L21) -- this script should print "Deploying Elasticsearch for dataset XX to project YY". I was in the wrong project and I didn't realize it.

Same for create-cluster.sh.